workers/unity/Assets/Playground/Scripts/UI/InitUIBehaviour.cs

```diff
@@ -0,0 +1,37 @@
+using Improbable.Gdk.Core;
+using Improbable.Gdk.Subscriptions;
+using Playground;
+using Playground.Scripts.UI;
+using UnityEngine;
+
+namespace Assets.Playground.Scripts.UI
```

Comment:
Should this have `Assets` prefixed to it?

Reply:
nop, fixed